sub-projects/hub/src/server/personofinterest/poi-router.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+
+const personManger = require('personofinterest/poi-manager.js');
```

Comment:
Consider using a shortened version of this folder name. `poi` would be unique and shorter